app/scripts/modules/amazon/src/loadBalancer/configure/choice/awsLoadBalancerChoice.modal.ts

```diff
@@ -11,32 +11,48 @@ class AwsLoadBalancerChoiceCtrl implements IController {
 
   constructor(private $uibModal: any,
               private $uibModalInstance: IModalInstanceService,
-              private application: Application) {
+              private application: Application,
+              private loadBalancer: any,
+              private isNew: boolean,
+              private forPipelineConfig: boolean) {
     'ngInject';
   }
 
   public $onInit(): void {
     this.choices = LoadBalancerTypes;
     this.choice = this.choices[0];
+    if (this.loadBalancer) {
+      // If we're editing an existing LB, preset the choice based on config,
+      const needed_type = this.loadBalancer.loadBalancerType;
+      this.choice = LoadBalancerTypes.filter(function(el) {
+        return el.type === needed_type;
+      })[0];
```

Comment:
Well, this will solve the camelcase problem, too, so I'll see if it works.  I was having trouble getting the scope to work (or at least, that's what I thought was the problem).